keepassxc-browser/content/totp-field.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const ignoreRegex = /(bank|coupon|postal|user|zip).*code|(en|de)code(d|r)*|comment|author|error/i;
+const ignoreRegex = /(bank|coupon|postal|user|zip|promo|promotional).*code|(en|de)code(d|r)*|comment|author|error/i;
```

Comment:
```suggestion
const ignoreRegex = /(bank|coupon|postal|user|zip|promo).*code|(en|de)code(d|r)*|comment|author|error/i;
```

Since promo is a prefix of promotional, this isn't even necessary! :-)